packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -512,28 +512,31 @@ contract BridgeFacet is BaseConnextFacet {
       // hash(canonicalId, canonicalDomain), this is safe even when the address(0) asset is not
       // whitelisted.
       bytes32 key;
+      TokenConfig memory config;
```

Comment:
this is... slightly problematic. We should probably switch to
```solidity
TokenConfig storage config;
```

So instead we're assigning a reference to a storage location, instead of copying ALL the values from the token config into memory (doing an SLOAD for all the tuple values). That way, we can pull values from storage ONLY as needed